torch/_inductor/cpu_vec_isa.py

```diff
@@ -317,8 +315,6 @@ def _check_and_append_supported_isa(
 @functools.lru_cache(None)
 def valid_vec_isa_list() -> List[VecISA]:
     isa_list: List[VecISA] = []
-    if sys.platform == "darwin" and platform.processor() == "arm":
-        isa_list.append(VecNEON())
```

Comment:
Why are you removing vectorization support for MacOS?

Reply:
I assumed adding platform.machine as "arm64" would enable vec for macOS, but apparently that does not resolve the macOS story. I'll revert the commit with this change. Thanks!